code/controllers/subsystem/SSticker.dm

```diff
@@ -829,22 +829,48 @@ SUBSYSTEM_DEF(ticker)
 			continue
 		.++
 
-/// Return whether or not a given biohazard is an active threat.
-/// For blobs, this is simply if there are any overminds left. For terrors and
-/// xenomorphs, this is whether they have overwhelming numbers.
-/datum/controller/subsystem/ticker/proc/biohazard_active_threat(biohazard)
+/// Record the initial time that a biohazard spawned, and then
+/// start the callback loop of recording its population according
+/// to the [BIOHAZARD_POP_INTERVAL].
+/datum/controller/subsystem/ticker/proc/record_biohazard_start(biohazard)
+	SSblackbox.record_feedback("associative", "biohazard_starts", 1, list("type" = biohazard, "time_ds" = world.time - time_game_started))
+	update_biohazard_pop_stats(biohazard)
+
+/datum/controller/subsystem/ticker/proc/update_biohazard_pop_stats(biohazard)
+	var/current_count = biohazard_count(biohazard)
+	SSblackbox.record_feedback("ledger", "biohazard_pop_[BIOHAZARD_POP_INTERVAL_STR]_interval", current_count, biohazard)
+	addtimer(CALLBACK(SSticker, TYPE_PROC_REF(/datum/controller/subsystem/ticker, update_biohazard_pop_stats), biohazard), BIOHAZARD_POP_INTERVAL)
```

Comment:
Any chance we can make a type of these to track on the SS itself as an assoc list list `last_tick_time[BIOHAZARD_POP_INTERVAL] = world.time`?

I dont like the idea of self-spawned timers, especially when this has no exit mechanism